providers/sendinblue/src/lib/sendinblue.provider.ts

```diff
@@ -51,9 +51,13 @@ export class SendinblueEmailProvider implements IEmailProvider {
       content: attachment?.file.toString('base64'),
       contentType: attachment.mime,
     }));
-    email.cc = options.cc?.map((ccItem) => ({ email: ccItem }));
-    email.bcc = options.bcc?.map((ccItem) => ({ email: ccItem }));
 
+    if (options.bcc)
+      email.bcc = options.bcc?.map((ccItem) => ({ email: ccItem }));
+
+    if (options.replyTo) {
+      email.replyTo.email = options.replyTo;
+    }
```

Comment:
This is duplicated. I guess you meant to do optional for `cc` right?

Reply:
Yes, updated!